_component_design/modal.md

```diff
@@ -101,9 +119,37 @@ Center modals vertically and horizontally in the browser window.
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Loading
+### Spacing
+
+Use the following spacing guidelines when you construct a modal.
+
+- Center-align modals vertically and horizontally
```

Comment:
As a sentence, this line needs a period at the end. However, it isn't parallel with the rest of the list. Suggestion:
```
- Alignment: Centered vertically and horizontally
```